.github/pull_request_template.md

```diff
@@ -26,8 +26,8 @@
 #### Documentation
 
 - [ ] Updated build docs
-- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
-- [ ] Updated field spec (https://docs.google.com/spreadsheets/d/1v1yqcSC54Vag4mxmTp8e6ZrnxITqzbjCA4A9XPQdXk8/edit?usp=sharing)
+- [ ] Updated editor guidelines (See https://torchbox.com/project-google-doc-links/ for the private link, for Torchbox employees only)
```

Comment:
Question: Any reason not to just use the intranet instead? Save the need for a password-protected page on our website? That feels like a better place for this information.

Reply:
that's a much better idea!

Reply:
I have created the intranet page and updated the links.